suite-common/connect-init/src/connectInitThunks.ts

```diff
@@ -62,6 +62,7 @@ export const connectInitThunk = createThunk(
             'backupDevice',
             'recoveryDevice',
             'checkFirmwareAuthenticity',
+            'authorizeCoinJoin',
```

Comment:
'getOwnershipProof', 'setBusy' are also used occasionally while running session

Reply:
probably does not hurt to add them too although I could not figure out any buttons that would need to be disabled

Reply:
scenario:
you are running CJ session in the backgroud, UI is unlocked because you are not in critical phase yet.
to registrer input into CJ round you need to provide ownershipProof of address, this operiation runs automatically without user interaction.

now you are in "receive" tab and you are going to to "showAddress" while in the same time "getOwnershipProof" is called in the background.
i believe that "show address" button is already covered by "is device locked" condition therefore it will be disabled for a short while

Reply:
yes, that should be it!